### ISSUE TITLE
Unable to use XOR-URL as mp4 resource 

### ISSUE THREAD

**Issue by hunterlester** [maidsafe/safe_app_nodejs#338]
While testing this [safe-patter issue](https://github.com/maidsafe/safe-patter-js/issues/12), I generated an XOR-URL, `safe://hygducftyhxpnrtc3t8zoqdpn1tguyuitpabs8grgez1qqojphfqbqqauxb9o`, for a 3.67MB mp4.
I'm unable to view it in browser, as already reported in [safe_browser #489](https://github.com/maidsafe/safe_browser/issues/489), however furthermore, I'm unable to use it as a `src` for a `video` tag at `safe://xor.resources`.
```
<html>
  <head></head>
  <body>
      <video controls>
          <source src="safe://hygducftyhxpnrtc3t8zoqdpn1tguyuitpabs8grgez1qqojphfqbqqauxb9o" type="video/mp4">
      </video>
  </body>
</html>
```
This `video` element will remain blank.

I am able to, however, use an NFS file as a `src` and view the video at `safe://home.resources`, like so:
```
<html>
  <head></head>
  <body>
      <video controls>
          <source src="safe://home.resources/test_video5.mp4" type="video/mp4">
      </video>
  </body>
</html>
```

I see multiple `-301: File not found` errors.
Probably a simple enhancement to fetching behavior, where it's currently solely looking for an NFS file resource instead of a raw buffer from an ImmutableData.
Although NFS files are not expected to have been ciphered, as ImmutableData is, so that does add a little more complexity to fetching.

**Comment by bochaco** [comment]
@hunterlester I don't think this is a limitation in NFS file vs. XOR-URL/ImmD resource, we are already using XOR-URLs in `src` tags in Patter and WebID Mgr. app, so the problem is more related to this particular type of file.
